packages/components/tree/src/tree-node-content.vue

```diff
@@ -24,11 +24,17 @@ export default defineComponent({
       const { data, store } = node
       return props.renderContent
         ? props.renderContent(h, { _self: nodeInstance, node, data, store })
-        : h('span', { class: ns.be('node', 'label') }, [
-            tree.ctx.slots.default
-              ? tree.ctx.slots.default({ node, data })
-              : node.label,
-          ])
+        : h(
```

Comment:
Don't change the whole here

Reply:
You just need to add `width: 100%`

Reply:
> You just need to add `width: 100%`

I'm worried about affecting other elements using this class name，so I just add the new className for this issue.

Reply:
Tested without any effect

Reply:
But this is not the point, I think this https://github.com/element-plus/element-plus/pull/9613/commits/7f1d3eb3cac80b8566ee0fcc2b21bfd8602685bf is problematic, it will create a new node, which affects performance, let us Let’s discuss this PR first, thank you for your PR

Reply:
> But this is not the point, I think this [7f1d3eb](https://github.com/element-plus/element-plus/commit/7f1d3eb3cac80b8566ee0fcc2b21bfd8602685bf) is problematic, it will create a new node, which affects performance, let us Let’s discuss this PR first, thank you for your PR

okay, I think the same as you